cmd/habitatctl/commands/start.go

```diff
@@ -90,7 +94,6 @@ var startCmd = &cobra.Command{
 
 func init() {
 	startCmd.PersistentFlags().StringVarP(&commName, "comm", "c", "", "name of community for which to start process")
-	startCmd.MarkPersistentFlagRequired("comm")
```

Comment:
so the reason I added this is because we want all commands to be within a specific community. is this not true in some cases?

Reply:
True this doesn't make sense anymore now that the raft app got refactored out of existence